smithy-aws-protocol-tests/model/restXml/document-structs.smithy

```diff
@@ -227,7 +227,6 @@ structure SimpleScalarPropertiesInputOutput {
 // This example serializes empty string in the top level XML document.
 @idempotent
 @http(uri: "/XmlEmptyStrings", method: "PUT")
-@tags(["client-only"])
```

Comment:
Looks like there's a set of `@httpResponseTests` for this operation. Should they have the `appliesTo` member added?